pkgs/development/libraries/physics/geant4/datasets.nix

```diff
@@ -57,20 +68,22 @@ in
       version = "5.2";
```

Comment:
Official web site advertises G4RadioactiveDecay5.3

Reply:
whoops, I did download 5.3, but because of the kerfluffle with the conflict between it and photonevaporation, I forgot to up the version number.  I'll force push a fix.